recipe_scrapers/thecookingguy.py

```diff
@@ -0,0 +1,117 @@
+# mypy: allow-untyped-defs
+import sys
+import warnings
+
+from ._abstract import AbstractScraper
+from ._exceptions import ElementNotFoundInHtml
+from ._grouping_utils import IngredientGroup
+from ._utils import get_yields, normalize_string
+
+if not sys.warnoptions:
+    warnings.simplefilter("ignore")
```

Comment:
Most recent commit should do this, I moved the if statement into the function. If no flags are set, then it will ignore the warning. If a flag is set, then it will follow the default behavior of that flag (as described in https://docs.python.org/3/library/warnings.html#the-warnings-filter). There's no more module level filtering now and I tested that by putting a temporary warning in the title() method and saw that, when run with no flags, the title warning printed as normal and the total_time warning was ignored.